packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -82,7 +82,6 @@ const CreateAdForm = () => {
             disposeApiErrorReaction();
             my_ads_store.setApiErrorMessage('');
             floating_rate_store.setApiErrorMessage('');
-            my_ads_store.setShowAdForm(false);
```

Comment:
This was added a long time ago to synchronize with cancel add payment method modal, but since now cancel add payment method modal accepts a prop to control this behaviour, this setup is no longer needed